vt2pbf/service/feature.py

```diff
@@ -78,6 +78,8 @@ def _encode_feature_geometry(self, raw_geometry: List[List[List[int]]]) -> List[
         x = 0
         y = 0
         result = []
+        # handle point feature type
+        raw_geometry = [raw_geometry] if self.feature_type == 1 else raw_geometry
```

Comment:
According to typehint in this function raw_geometry must have a structure like "List[List[List[int]]]"
It means you cannot just wrap raw_geometry once more cause of feature_type.

I sure the problem should be resolved before this function, namely in add_geometry. Also add_geometry typehints must be changed for more basic structure to support Poin geometry

Reply:
I u want u can make this changes and I will merge PR and update library version, if not - I will do it sometimes